content/en/seachat/getting-started/02-quick-start.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Quick Start"
+description: "SeaChat Intro and getting started"
+lead: ""
+date: 2024-03-08 10:43:51.069 +0100
+lastmod: 2024-03-08 10:43:51.069 +0100
+weight: 202
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/02-quick-start/
+  - /seachat/seachat-intro/02-quick-start/
+---
+
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
+* This step-by-step guide will walk you through creating your first SeaChat agent from scratch. In just under 10 minutes, you'll have your own customized AI agent set up and ready to deploy on your website.
+* For more detailed guidance, please consult the [SeaChat User Manual](/en/seachat/seachat-intro/02-how-to-create/). Should you encounter any issues or have questions, our [support team](#support) is on standby to assist you. Don't hesitate to reach out.
+
+---
+## Step 1: Create a Workspace 
+If this is your first time logging into SeaChat, you will have to create a workspace to kickstart the process. Enter the name of your workspace and let’s begin!
+
+## Step  2: Your First SeaChat Agent
+Click on **Add New Agent** to configure your new agent. If you already have a workspace, you can click on the **My Agent** in the top-right corner to create a new agent.
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step2.png" alt="">
+<br/>
+
+<br/>
+
+Once you have picked the right name for your agent, move on to **Description** to give your agent some context of its role and functions before you choose its use case and response language.
+<br/>
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step2-2.png" alt="">
+<br/>
+
+<br/>
+
+> :bulb: **Note** 
+>
+>  The live agent feature will require a human agent in case the customer wishes to be transferred to one. The user will then have the option to request a human agent during the chat.
```

Comment:
this is a good note. 

Can you also clarify why we use "Agent" but not "bot"? and when we say "Agent" did we mean human agent or AI agent?